lib/game.rb

```diff
@@ -0,0 +1,72 @@
+require_relative 'frame'
+
+class Game
+  attr_reader :frames
+  
+  def initialize
+    @frames = [Frame.new]
+  end
+
+  def roll(pins)
+    case 
+    when frame_count == 1 && current_frame.in_play
+         current_frame.knocked(pins)
```

Comment:
Could we combine this case with the `frame_count <=8 && current_frame.in_play` scenario?
This would mean amending `last_frame_strike?` or `last_frame` to handle when the `frame_count` is 1, but would mean one fewer case